### ISSUE TITLE
Requirements update

### ISSUE THREAD

**Issue by giovannipds** [netdragoon/zipcode#3]
Hi there. I'm needing `guzzlehttp/guzzle:6.3.*` in a project and I'm not being able to update 'cause of these requirements. Any chances of updating this package and supporting the major version here? Like - for yesterday? :laughing: Thanks in advance (I hope it'll be better for us all)!

**Comment by giovannipds** [comment]
```sh
λ composer outdated
guzzlehttp/guzzle            5.3.3   6.3.3   Guzzle is a PHP HTTP client libra...
laravel/framework            v5.5.45 v5.7.23 The Laravel Framework.
nesbot/carbon                1.36.2  2.10.1  A simple API extension for DateTime.
phpdocumentor/type-resolver  0.4.0   0.7.1
phpspec/php-diff             v1.0.2  v1.1.0  A comprehensive library for gener...
phpspec/phpspec              2.5.8   5.1.0   Specification-oriented BDD framew...
phpunit/php-code-coverage    2.2.4   6.1.4   Library that provides collection,...
phpunit/php-file-iterator    1.4.5   2.0.2   FilterIterator implementation tha...
phpunit/php-timer            1.0.9   2.0.0   Utility class for timing
phpunit/php-token-stream     1.4.12  3.0.1   Wrapper around PHP's tokenizer ex...
phpunit/phpunit              4.8.36  7.5.2   The PHP Unit Testing framework.
phpunit/phpunit-mock-objects 2.3.8   6.1.2   Mock Object library for PHPUnit
sebastian/comparator         1.2.4   3.0.2   Provides the functionality to com...
sebastian/diff               1.4.3   3.0.1   Diff implementation
sebastian/environment        1.3.8   4.0.2   Provides functionality to handle ...
sebastian/exporter           1.2.2   3.1.0   Provides the functionality to exp...
sebastian/global-state       1.1.1   2.0.0   Snapshotting of global state
sebastian/recursion-context  1.0.5   3.0.0   Provides functionality to recursi...
sebastian/version            1.0.6   2.0.1   Library that helps with managing ...
symfony/console              v3.4.21 v4.2.2  Symfony Console Component
symfony/debug                v3.4.21 v4.2.2  Symfony Debug Component
symfony/event-dispatcher     v3.4.21 v4.2.2  Symfony EventDispatcher Component
symfony/finder               v3.4.21 v4.2.2  Symfony Finder Component
symfony/http-foundation      v3.4.21 v4.2.2  Symfony HttpFoundation Component
symfony/http-kernel          v3.4.21 v4.2.2  Symfony HttpKernel Component
symfony/process              v3.4.21 v4.2.2  Symfony Process Component
symfony/routing              v3.4.21 v4.2.2  Symfony Routing Component
symfony/var-dumper           v3.4.21 v4.2.2  Symfony mechanism for exploring a...
symfony/yaml                 v3.4.21 v4.2.2  Symfony Yaml Component
vlucas/phpdotenv             v2.6.1  v3.3.1  Loads environment variables from ...
```

**Comment by giovannipds** [comment]
To anyone needing an updated version, this [fork](https://github.com/danilolutz/zipcode)'s a little more updated (**update: disregard this if this issue's closed**)

**Comment by giovannipds** [comment]
I've sent a PR updating that requirement.

**Comment by giovannipds** [comment]
Ping @netdragoon.

**Comment by giovannipds** [comment]
Seems you're inactive. I'll ping an old contributor to see if there's any chance of he helping me to contact you: @vluzrmos.

**Comment by fulviocanducci** [comment]
@giovannipds 
I'll check, wait.

**Comment by netdragoon** [comment]
New version released v2.0.1 without dependence package Guzzle

https://github.com/netdragoon/zipcode/releases/tag/v2.0.1

**Comment by netdragoon** [comment]
Update https://github.com/netdragoon/zipcode/releases/tag/v2.0.2
